src/components/Forum/types.ts

```diff
@@ -14,7 +14,7 @@ export interface DiscussionContent {
   master?: string | null;
   replyTo?: string | null;
   mentions?: string[];
-  data?: Record<string, string>;
+  data?: any;

```

Comment:
Can we get types for this or is too hard?

Reply:
The type on orbis is actually just anything but maybe we can write our own in future. Will leave it as any for now though